connector/azure_storage/src/main/java/it/pagopa/selfcare/party/connector/azure_storage/model/AnacDataTemplate.java

```diff
@@ -22,4 +24,11 @@ public class AnacDataTemplate implements Station {
     private boolean anacEngaged;
     @CsvBindByName(column = "ANAC_abilitato")
     private boolean anacEnabled;
+
+    public String getTaxCode() {
+        if(this.taxCode.length() < 11) {
+            return String.join("", Collections.nCopies(11 - taxCode.length(), "0")) + this.taxCode;
```

Comment:
It might be simply using StringUtils.leftPad of commons-lang3,,,look at https://www.baeldung.com/java-pad-string